snippets/base/templates/base/includes/snippet_as.css

```diff
@@ -18,6 +19,14 @@
   }
 }
 
+#snippets-container a {
+  color: var(--newtab-link-primary-color);
```

Comment:
You could make whatever `--newtab-link-primary-color` evaluates to the fallback, if design is OK with it

Reply:
The way it is now though it will just cacade up like before if the variable doesn't exist.